lightshow/database.py

```diff
@@ -148,11 +161,28 @@ def from_materials_project(cls, **kwargs):
         except KeyError:
             pass
 
-        with MPRester(api_key) as mpr:
```

Comment:
Why did you get rid of the context manager `with` statement here? This cleans up `mpr` after use and we should use it.

Reply:
You are correct now it is changed